code/datums/traits/negative.dm

```diff
@@ -0,0 +1,180 @@
+	value = -1
+	medical_record_text = "Patient has a persistent, genuine phobia of arachnids."
+
+/datum/trait/arachnophobia/add()
+	var/mob/living/carbon/human/H = trait_holder
+	H.gain_trauma(/datum/brain_trauma/mild/phobia, TRUE, "spiders")
+
+
+
+/datum/trait/heavy_sleeper
+	name = "Heavy Sleeper"
+	desc = "You sleep like a rock! Whenever you're put to sleep, you sleep for a little bit longer."
+	value = -1
+	mob_trait = TRAIT_HEAVY_SLEEPER
+	gain_text = "<span class='danger'>You feel sleepy.</span>"
+	lose_text = "<span class='notice'>You feel awake again.</span>"
+	medical_record_text = "Patient has abnormal sleep study results and is difficult to wake up."
+
+
+
+/datum/trait/nearsighted
+	name = "Nearsighted"
+	desc = "You are nearsighted without prescription glasses, but spawn with a pair."
+	value = -1
+	mob_trait = TRAIT_NEARSIGHT
+	gain_text = "<span class='danger'>Things far away from you start looking blurry.</span>"
+	lose_text = "<span class='notice'>You start seeing faraway things normally again.</span>"
+	medical_record_text = "Patient requires prescription glasses in order to counteract nearsightedness."
+
+/datum/trait/nearsighted/on_spawn()
+	var/mob/living/carbon/human/H = trait_holder
+	var/obj/item/clothing/glasses/regular/glasses = new(get_turf(H))
+	H.put_in_hands(glasses)
+	H.equip_to_slot(glasses, slot_glasses)
+	H.regenerate_icons() //this is to remove the inhand icon, which persists even if it's not in their hands
+
+
+
+/datum/trait/nonviolent
+	name = "Pacifist"
+	desc = "The thought of violence makes you sick. So much so, in fact, that you can't hurt anyone."
+	value = -2
+	mob_trait = TRAIT_PACIFISM
+	gain_text = "<span class='danger'>You feel repulsed by the thought of violence!</span>"
+	lose_text = "<span class='notice'>You think you can defend yourself again.</span>"
+	medical_record_text = "Patient is unusually pacifistic and cannot bring themselves to cause physical harm."
+
+/datum/trait/nonviolent/on_process()
+	if(trait_holder.mind && trait_holder.mind.antag_datums.len)
+		to_chat(trait_holder, "<span class='boldannounce'>Your antagonistic nature has caused you to renounce your pacifism.</span>")
+		qdel(src)
+
+
+
+/datum/trait/poor_aim
+	name = "Poor Aim"
+	desc = "You're terrible with guns and can't line up a straight shot to save your life. Dual-wielding is right out."
+	value = -1
+	mob_trait = TRAIT_POOR_AIM
+	medical_record_text = "Patient possesses a strong tremor in both hands."
+
+
+
+/datum/trait/prosopagnosia
+	name = "Prosopagnosia"
+	desc = "You have a mental disorder that prevents you from being able to recognize faces at all."
+	value = -1
+	mob_trait = TRAIT_PROSOPAGNOSIA
+	medical_record_text = "Patient suffers from prosopagnosia and cannot recognize faces."
+
+
+
+/datum/trait/prosthetic_limb
+	name = "Prosthetic Limb"
+	desc = "An accident caused you to lose one of your limbs. Because of this, you now have a random prosthetic!"
+	value = -1
```

Comment:
I mean, bonus points if you can pick WHICH limb you're missing.

Reply:
It's doable, but for now this is fine to me.